bin/scraper/projects.json

```diff
@@ -0,0 +1 @@
re sensor) with a modern imperative language. We introduce first-class primitives for sensor[…]","url":"https://projects.eclipse.org/projects/iot.mita","logo":"https://projects.eclipse.org/sites/default/files/logo_3.png","website":null,"state":"Incubating","provisioned":true},"iot.hip":{"name":"Eclipse HIP","id":"iot.hip","description":"Through HIP, localised processing or preperation of data is intended to reduce the connectivity and central processing requirements within IOT - much like the contemporary Fog computing paradigm, but without the need for sophisticated edge computing[…]","url":"https://projects.eclipse.org/projects/iot.hip","logo":null,"website":"http://www.eclipse.org/hip","state":"Incubating","provisioned":true},"iot.fog05":{"name":"Eclipse fog05","id":"iot.fog05","description":"Early IoT applications, especially those addressing the&nbsp;&nbsp;consumer market, have been embracing cloud-centric architectures in which data is pushed up to the cloud.&nbsp;It is within&nbsp;the cloud the everything takes place before eventually[…]","url":"https://projects.eclipse.org/projects/iot.fog05","logo":null,"website":null,"state":"Incubating","provisioned":true},"iot.tahu":{"name":"Eclipse Tahu","id":"iot.tahu","description":"Eclipse Tahu addresses the existence of legacy SCADA/DCS/ICS protocols and infrastructures and provides a much-needed definition of how best to apply MQTT into these existing industrial operational environments.\r\n\r\nBasic network[…]","url":"https://projects.eclipse.org/projects/iot.tahu","logo":null,"website":null,"state":"Incubating","provisioned":true},"iot.volttron":{"name":"Eclipse VOLTTRON","id":"iot.volttron","description":"The Eclipse VOLTTRON software platform is Linux-based and capable of running on small, single-board computers as well as in the cloud. It deploys applications known as V-agents that gain access to energy data via automation systems, identifying and[…]","url":"https://projects.eclipse.org/projects/iot.volttron","logo":null,"website":"https://volttron.org/","state":"Incubating","provisioned":true},"iot.keyple":{"name":"Eclipse Keyple","id":"iot.keyple","description":"The goal of Eclipse Keyple is to allow developers to easily implement fast and secure off-line contactless transactions (using NFC cards, mobile phones, &hellip;) based on the Calypso standard.\r\n\r\nMore specifically, Keyple is a set of open source[…]","url":"https://projects.eclipse.org/projects/iot.keyple","logo":null,"website":null,"state":"Incubating","provisioned":true},"iot.yasham":{"name":"Eclipse Yasham","id":"iot.yasham","description":"Eclipse Yasham is an&nbsp;Open &amp; Extensible Cloud-Native Smart Living Platform for Consumer IoT.\r\n\r\n\r\n\r\nSome features at a glance\r\n\r\n\r\n\tServer Side API Gateway (REST API)\r\n\r\n\t\r\n\t\tService Registration and discovery\r\n\t\tCaching and[…]","url":"https://projects.eclipse.org/projects/iot.yasham","logo":null,"website":null,"state":"Incubating","provisioned":false}}
```

Comment:
Please make sure that this is the original file.

Reply:
Done